crates/common/mqtt_channel/src/tests.rs

```diff
@@ -454,9 +442,8 @@ mod tests {
     }
 
     #[tokio::test]
-    #[serial]
     async fn ensure_that_all_messages_are_sent_before_disconnect() -> Result<(), anyhow::Error> {
-        let broker = mqtt_tests::test_mqtt_broker();
```

Comment:
Using individual ports does not scale because of a number of tests. That's the reason why we used a single port and ran them in a serial manner.

Reply:
What do you mean with "does not scale"?

Reply:
I mean having individual ports per test. We might not have that many free ports.

Reply:
In linux, dynamic ports are 49152 up to 65535 - that's 16383 ports we can use. I would say it suffices for the next few years, don't you think? :laughing: